### ISSUE TITLE
Remove special code for `a11y`, `actions` and `links` in `@storybook/server`

### ISSUE THREAD

**Issue by tmeasday** [storybookjs/storybook#9741]
Ideally the user could use these addons in the "normal" way without `@storybook/server` needing to do anything. This might mean some tweaks about what the "normal" way is:

- [x] `a11y` - install the decorator globally by default (in a preset), ensure it is easy to enable or disable in a story with parameters.

- [x] `links` - install the decorator globally in the preset

- [x] `actions` - install the "event" decorator globally in a preset, ensure it is configurable via parameters (and a no-op by default).

**Comment by jonspalmer** [comment]
@tmeasday Trying to understand what you are expecting in terms of "installing in the preset" are there examples of that approach?

**Comment by tmeasday** [comment]
Looks like you figured it out (https://github.com/storybookjs/storybook/pull/9932) -- or is there more you need to know?

**Comment by jonspalmer** [comment]
I think I figured it out  in the other PRs. Thanks for the followup

**Comment by tmeasday** [comment]
Is this task now done?

**Comment by jonspalmer** [comment]
I think we still want to merge it.

**Comment by shilman** [comment]
Merged. Actions with args should be a slam dunk to round this out.

**Comment by shilman** [comment]
Ta-da!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.0.0-alpha.28 containing PR #9931 that references this issue. Upgrade today to try it out!

You can find this prerelease on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.

**Comment by shilman** [comment]
Olé!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.0.0-alpha.30 containing PR #9933 that references this issue. Upgrade today to try it out!

You can find this prerelease on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.
